.github/workflows/ci.yml

```diff
@@ -211,13 +218,19 @@ jobs:
 
   pre-merge-checks-complete:
     if: ${{ always() && github.event_name == 'pull_request' && github.event.pull_request.merged == false }}
-    needs: [build-linux-x86_64-pre-merge, local-environment-tests, devshell-tests, upload-chain-specs-pre-merge, documentation-tests]
+    needs: [
+      build-linux-x86_64-pre-merge,
+      local-environment-tests-alert,
+      devshell-tests,
+      upload-chain-specs-pre-merge,
+      documentation-tests
+    ]
     runs-on: ubuntu-latest
     steps:
       - name: Check if any needed job failed
         run: |
           if [[ "${{ needs.build-linux-x86_64-pre-merge.result }}" != "success" ||
-                "${{ needs.local-environment-tests.result }}" != "success" ||
+                "${{ needs.local-environment-tests-alert.result }}" != "success" ||
```

Comment:
I think alert step will succeed with failed tests, we should require tests step to succeed